lib/content-services/src/lib/security/services/models/security-controls-group-info-response.interface.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SecurityMark } from '@alfresco/js-api';
+
+export interface SecurityControlsGroupInfoResponse {
+    id: string;
+    groupName: string;
+    groupType: string;
```

Comment:
I have question regarding to that groupType field. Can it be any string or do we have limited set of allowed types? If it is limited then maybe we can create something like enum as a type here?

Reply:
I found such group types:  _HIERARCHICAL_, _USER_REQUIRES_ANY_ and `USER_REQUIRES_ALL` types, but I am not 100% sure if we are limited only for those ones.

Reply:
ok then leave it as it is if you are not sure to be safer or create enum with these 3 types and somebody can eventually update it later if there will be more types but treat that as optional so you can leave it as it is now.